Replicator/Pusher.hh

```diff
@@ -45,13 +46,36 @@ namespace litecore { namespace repl {
         // Checks if a given document should be filtered by this pusher.  The revision body
         // must be loaded prior to this call.
         bool documentShouldBeFiltered(C4Document* doc) const {
```

Comment:
I find it confusing that this method's return value has the opposite meaning from the filter functions. Could you reverse it to be consistent? I.e. return true to use the doc, false to skip it.